externs/olx.js

```diff
@@ -3547,7 +3598,7 @@ olx.layer.VectorOptions.prototype.renderBuffer;
 
 /**
  * Source.
- * @type {ol.source.Vector}
+ * @type {ol.source.Vector|ol.source.VectorTile}
```

Comment:
I think removal makes sense (and adapting the existing example to what you propose), it will only confuse users otherwise on what to use.